src/sentry/api/event_search.py

```diff
@@ -103,7 +104,7 @@
 text_in_filter = negation? text_key sep text_in_list
 
 # standard key:val filter
-text_filter = negation? text_key sep search_value
+text_filter = negation? text_key sep operator? search_value
```

Comment:
I can do it if you want, but would you be able to update the frontend grammar to also support this operator in this PR. It should be self explanatory. Find the `.pegjs` file